marketplace/deployer_helm_base/Dockerfile

```diff
@@ -14,15 +14,15 @@ RUN pip3 install \
       pyyaml \
       six
 
-RUN for full_version in 1.15.12 1.16.15 1.17.14 1.18.12;  \
+RUN for full_version in 1.20.12 1.21.6 1.22.3;  \
      do \
         version=${full_version%.*} \
         && mkdir -p /opt/kubectl/$version \
         && wget -q -O /opt/kubectl/$version/kubectl \
             https://storage.googleapis.com/kubernetes-release/release/v$full_version/bin/linux/amd64/kubectl \
         && chmod 755 /opt/kubectl/$version/kubectl; \
      done;
-RUN ln -s /opt/kubectl/1.16 /opt/kubectl/default
+RUN ln -s /opt/kubectl/1.20 /opt/kubectl/default
 
 RUN mkdir -p /bin/helm-downloaded \
     && wget -q -O /bin/helm-downloaded/helm.tar.gz \
```

Comment:
Does it make sense to update the helm version as well?
(Not sure what changes have been made)